### ISSUE TITLE
[Bug - TBCC] Characters not show in the Professions Window

### ISSUE THREAD

**Issue by virtualbjorn** [stpain/guildbook#60]
The issue is self-explanatory. characters doesn't show up in the Characters listing

![image](https://user-images.githubusercontent.com/19782129/132243422-9af89bb1-3cd2-428f-ab70-5b67b18d1200.png)

**Comment by stpain** [comment]
I'm assuming you have clicked on the Blacksmith button on the left as this is a list of blacksmith crafts, clicking an item in the list should then show a list of who can craft that item.

**Comment by virtualbjorn** [comment]
Can we have the characters listed by default?

So people would know right away which characters can do BS and clicking the said Character will focus the view on what that certain character have.

**Comment by stpain** [comment]
you can view a characters tradeskill items by clicking the tradeskill icon on the roster listview

**Comment by virtualbjorn** [comment]
Yeah, that's what I've been doing before. But I'm just bugged by an empty characters listing on the right side of the UI.  I'm hoping the characters with the said profession will be listed by default. Thank you.

**Comment by stpain** [comment]
well if you click a characters profession from the roster then you wont see any characters as its the recipes for the character you clicked.
To view characters you'd need to select a recipe item.

So you can 
Click profession >Click recipe item > see characters
Click roster profession button for character A > view recipes character A has

I will look at adding all blacksmiths when you click the blacksmith button BUT that wont mean all blacksmiths can craft ALL items, hence why you click an item to see who can craft it. To see all blacksmiths you can sort the roster view by clicking the trade column header button.

**Comment by virtualbjorn** [comment]
I'm okay with seeing all the crafters for the certain profession in the guild in the characters listing after clicking the profession.
Just additional information if you chose to add the requested option above, I think it is also good to be able to click on that certain character and show only that character's recipe.

**Comment by stpain** [comment]
well i can add it in easily enough as the function already exists, it'll just be a case of having the same function called from 2 places, either from the prof icons on the roster listview or from the characters list on the prof tab.

so you want to be able to
open tradeskill tab > click blacksmith and see all blacksmith recipes and also all characters with blacksmithing > click a recipe to filter the characters to only those with the recipe -OR- click a character to view only their recipes

as well as being able to view a specific characters recipes by clicking their prof icon in the roster listview, which is exactly what the above would do and already exists?

![tradeskill views](https://user-images.githubusercontent.com/47498931/132716692-3f328f66-07e3-4c71-bc0f-d7a7b700907d.gif)

**Comment by virtualbjorn** [comment]
So we always have to navigate to the roster to only show the recipes of a certain character?
